deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -1847,28 +1843,90 @@ class ChatViewController: UITableViewController, UITableViewDropDelegate {
 
     @available(iOS 13.0, *)
     override func tableView(_ tableView: UITableView, previewForHighlightingContextMenuWithConfiguration configuration: UIContextMenuConfiguration) -> UITargetedPreview? {
-        return makeTargetedPreview(for: configuration)
+        return targetedPreview(for: configuration, reactionsHidden: false)
     }
 
     @available(iOS 13.0, *)
     override func tableView(_ tableView: UITableView, previewForDismissingContextMenuWithConfiguration configuration: UIContextMenuConfiguration) -> UITargetedPreview? {
-        return makeTargetedPreview(for: configuration)
+        return targetedPreview(for: configuration, reactionsHidden: true)
     }
 
     @available(iOS 13.0, *)
-    private func makeTargetedPreview(for configuration: UIContextMenuConfiguration) -> UITargetedPreview? {
+    private func targetedPreview(for configuration: UIContextMenuConfiguration, reactionsHidden: Bool) -> UITargetedPreview? {
         guard let messageId = configuration.identifier as? NSString else { return nil }
         guard let index = messageIds.firstIndex(of: messageId.integerValue) else { return nil }
         let indexPath = IndexPath(row: index, section: 0)
+        let message = dcContext.getMessage(id: messageId.integerValue)
+
+        guard let cell = tableView.cellForRow(at: indexPath) as? BaseMessageCell,
+              let messageSnapshotView = cell.messageBackgroundContainer.snapshotView(afterScreenUpdates: false) else { return nil }
+
+        cell.messageBackgroundContainer.isHidden = (reactionsHidden == false)
```

Comment:
are you avoiding `!reactionsHidden` because the `!` if is easy to overlook?  

(i personally find the standard boolean negation more readable and have rarely overseen it :)

Reply:
> because the ! if is easy to overlook?

Exactly!